android/expoview/src/main/java/host/exp/exponent/notifications/PushNotificationHelper.java

```diff
@@ -258,6 +313,21 @@ public Intent provide() {
     });
   }
 
+  private Bitmap loadRemoteImage(String imageURL, Context context) {
+    Bitmap imageBitmap = null;
+    try {
+      imageBitmap = Picasso.with(context).load(imageURL).get();
+    } catch (IOException ie) {
+      // It means the image is not loaded correctly.
+      // Do nothing.
```

Comment:
same here, let's log both these exceptions so that we provide the developer with as much debug information as possible